backend/model/model.go

```diff
@@ -929,6 +932,22 @@ type ErrorGroup struct {
 	Viewed         *bool   `json:"viewed"`
 }
 
+type ErrorGroupEventType string
+
+const (
+	ErrorGroupResolvedEvent ErrorGroupEventType = "ErrorGroupResolved"
+	ErrorGroupIgnoredEvent  ErrorGroupEventType = "ErrorGroupIgnored"
+	ErrorGroupOpenedEvent   ErrorGroupEventType = "ErrorGroupOpened"
+)
+
+type ErrorGroupActivityLog struct {
+	Model
+	ErrorGroupID int `gorm:"index"`
+	AdminID      int // when this is 0, it means the system generated the event
+	Admin        *Admin
+	EventType    ErrorGroupEventType
```

Comment:
Not included here is the snoozed event. Will be interested to know if folks have thoughts on how generic we want this.

For example: I named this `EventType` such that it could in theory handle updating things besides the state. For snoozing, the action is state=OPEN and snoozedUntil=time. 

If we want to keep this generic, we should probably add a new column called `EventParams` that's a JSON blob. The downside of this is that it's not typed. 

Alternatively, we can take the less overengineered approach and just name these columns `UpdatedState` and `UpdatedSnoozedUntil` if we don't think this will get a lot more usage.

Reply:
my instinct is to keep this less generic. i'd add another column like `UpdatedSnoozedUntil` that would be null when the event is not a snooze.

Reply:
What about error boundary feedback? That seem like we would want to store that in this table, yeah?

Reply:
I've added a JSONB column to store arbitrary data:
8041d321901b1d485d4c58440b349e6b5e6aece7